### PR TITLE
[MINI-5423] Revamp Unit Tests

### PR DESCRIPTION
**Pull request by laubengaier**
# Description
Revamp Unit Tests (Road to 80%)

## Links
https://jira.rakuten-it.com/jira/browse/MINI-5433

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by GondiTilak — COMMENTED** [review]
LGTM

**Review by Climbatize — APPROVED** [review]

File: Tests/Unit/MiniAppSecureStorageTests.swift [hunk 1]
```diff
@@ -283,7 +283,7 @@ class MiniAppSecureStorageTests: XCTestCase {
             }
         }
 
-        wait(for: [removeExpectation], timeout: 10.0)
+        wait(for: [removeExpectation], timeout: 30.0)
```
Comment (rleojoseph) [inline Tests/Unit/MiniAppSecureStorageTests.swift:286]: Any reason to increase the timeout? I think already 10 seems to be a bigger timeout. Also, we don't have any network APIs here.

**Review by rleojoseph — APPROVED** [review]
Looks good to me :+1:. Just one concern, please check it

File: Tests/Unit/MiniAppSecureStorageTests.swift (hunk 1 above)
Comment (laubengaier) [reply Tests/Unit/MiniAppSecureStorageTests.swift:286]: because of the bitrise downgrade

**Pull request merged**